com.unity.render-pipelines.core/Editor/Volume/VolumeComponentEditor.cs

```diff
@@ -222,12 +204,13 @@ static void ReloadDecoratorTypes()
         /// </summary>
         public new void Repaint()
         {
-            base.Repaint();
-            //if (m_Inspector != null) // Can happen in tests.
-                //m_Inspector.Repaint();
+            inspector?.Repaint();
+
             // Volume Component Editors can be shown in the ProjectSettings window (default volume profile)
             // This will force a repaint of the whole window, otherwise, additional properties highlight animation does not work properly.
             SettingsService.RepaintAllSettingsWindow();
+
+            base.Repaint();
```

Comment:
Looks like you've effectively enabled the code that was commented, could you open a little what happened here? Also the order of repaints is now different.